### PR TITLE
Replace the profiles and templates api implementations with grpc call into on-taskgraph

### PR DESCRIPTION
**Pull request by tannoa2**
Signed-off-by: Kranti Uppala <kranti.uppala@dell.com>
jenkins: ignore

### CODE REVIEW

**Comment by tannoa2** [comment]
jenkins: depends on https://github.com/RackHD/on-build-config/pull/339
jenkins: depends on https://github.com/RackHD/on-taskgraph/pull/289
jenkins: depends on https://github.com/RackHD/on-core/pull/308

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

File: lib/services/taskgraph-api-service.js [hunk 1]
```diff
@@ -106,6 +106,7 @@ function taskgraphApiServiceFactory(
                     }
                     resolve(response);
                 });
+                //args[0].req = "111"
```
Comment (geoff-reid) [inline lib/services/taskgraph-api-service.js]: remove this line

**Pull request merged**